### PR TITLE
Update cockpitpcpmetrics.c

### PR DESCRIPTION
**Pull request by fche**
Note that PM_CONTEXT_LOCAL is not quite what it sounds like.

### CODE REVIEW

File: src/bridge/cockpitpcpmetrics.c [hunk 1]
```diff
@@ -535,7 +535,7 @@ cockpit_pcp_metrics_prepare (CockpitChannel *channel)
       type = PM_CONTEXT_ARCHIVE;
       name = source;
     }
-  else if (g_str_equal (source, "system"))
+  else if (g_str_equal (source, "system")) // NB: this is for pure shared-library PMDAs; limited
```
Comment (stefwalter) [inline src/bridge/cockpitpcpmetrics.c:538]: That's what we want to consume for most of the graphs throughout Cockpit ... we don't want to have to bring up a whole other daemon just to display graphs. We want that those samples to be gathered within the cockpit-bridge process.

Comment (fche) [inline src/bridge/cockpitpcpmetrics.c:538]: Yes, that'll work, assuming that all the metrics you need are available via the DSO pmdas, and with the privileges of your -bridge process.  (Might want to rename "system" to "dso" or "local" or "direct" or something.)

Comment (stefwalter) [inline src/bridge/cockpitpcpmetrics.c:538]: @mvollmer yes "direct" is a good idea.

File: src/bridge/cockpitpcpmetrics.c [hunk 2]
```diff
@@ -543,7 +543,7 @@ cockpit_pcp_metrics_prepare (CockpitChannel *channel)
   else if (g_str_equal (source, "pmcd"))
     {
       type = PM_CONTEXT_HOST;
-      name = "127.0.0.1:44321";
+      name = "local:"; // prefer AF_UNIX, falls back to localhost
```
Comment (stefwalter) [inline src/bridge/cockpitpcpmetrics.c:546]: @mvollmer This is good too ... we should always prefer AF_UNIX.

File: src/bridge/cockpitpcpmetrics.c (hunk 1 above)
Comment (mvollmer) [inline src/bridge/cockpitpcpmetrics.c:538]: @fche, this comment is for you, I think.

Comment (stefwalter) [inline src/bridge/cockpitpcpmetrics.c:538]: Well I CC'd this comment to you since you now own the branch.

Comment (mvollmer) [inline src/bridge/cockpitpcpmetrics.c:538]: Oh, ok.

**Pull request closed without merging**